client/src/modules/vx-timer/store/actions.ts

```diff
@@ -5,9 +5,9 @@ export const createAndStartTimer = ({ commit }, opts: TimerOptions) => {
   commit(types.CREATE_AND_START_TIMER, opts)
 }
 
-export const createPromiseTimer = ({ commit }, opts: TimerOptions) => {
-  commit(types.CREATE_PROMISE_TIMER, opts)
-}
+// export const createPromiseTimer = ({ commit }, opts: TimerOptions) => {
+//   commit(types.CREATE_PROMISE_TIMER, opts)
+// }
```

Comment:
For future use?

Reply:
Yea, maybe. It's not used now but someone wrote it for a reason.